doc/source/reference/arrays.indexing.rst

```diff
@@ -1,5 +1,9 @@
 .. _arrays.indexing:
 
+.. for doctests
+   >>> import numpy as np
```

Comment:
CircleCI is failing due to the `.. arrays.indexing:` being in the wrong spot now. My guess is that the for doctest must be below the Indexing title? (or above the `.. arrays.indexing:`).

Reply:
I canceled the CI runs that had not completed. @pdebuyl please run the refguide-check --rst test locally before pushing.

Reply:
@mattip I did run refguide_check. Here, it is the sphinx build that errors, refguide_check did not complain.

Reply:
oops, sorry. I should have checked more carefully.

Reply:
no worry :-)